api/live.js

```diff
@@ -166,9 +143,37 @@ async function handleCheckin(userId, req, res) {
         .json({ invalid: `User has status ${profileStatus}` });
     }
 
+    const [pointsProfile, isCreated] = await models.Person.findOrCreate({
+      where: { identityId: userId },
+      defaults: { isBattlepassComplete: false }
+    });
+
+    if (!isCreated) {
+      return res
+        .status(400)
+        .json({ error: "Hacker has already had a person profile created" });
+    }
+
+    const minHouse = await models.House.findOne({
+      attributes: [
+        ["id", "id"],
+        ["name", "name"],
+        [
+          sequelize.literal(
+            "(SELECT COUNT(*) FROM persons where persons.houseId = House.id)"
+          ),
+          "personCount"
+        ]
+      ],
```

Comment:
Some spooky raw SQL, because sequelize absolutely butchers it if you depend on their abstractions

Reply:
Mostly has to do with it squashing other attributes on the houseModel, so all we can really extract is the personCount without dropping into raw SQL. Will monitor this closely, but I feel pretty confident about it, tested it quite a bit